### ISSUE TITLE
Feature Request: set path for junit xml output

### ISSUE THREAD

**Issue by mayl** [cpputest/cpputest#1446]
Currently, I don't see a way to direct the output xml file from the test runner when generating junit files with `-ojunit`.  #959 seems to be the closest issue I could find, but that user's issue seemed to be resolved by inserting prefixes into the output name with `-k`.  However, I would like to be able to actually specify an output path and/or emit the junit xml to standard output.  Does such a feature already exist and I've missed it?  If not, would there be any objections to someone adding this feature?

**Comment by basvodde** [comment]
I just ran a quick test and the XML files are written in current dir. That means you ought to be able to achieve this reasonably easy through: cd dir/I/want/output/in && build/dir/cpputest.

Not ideal, so I'll let the issue open... but not seem urgent, right?

**Comment by mayl** [comment]
Yes, we've been doing more or less exactly what you describe in a `bash` script (`cd` into the desired directory and run the test runner there).  Our build system is cross platform and can run the compiled test runner, but is not designed to `cd` to a directory first - hence the bash script.  Obviously, there are many possible workarounds but if CppUTest test runner could directly output to the desired directory through a flag, it would sidestep a lot of extra complication.

Thanks for leaving the issue open, it's at least good to have confirmation that the feature would not be rejected on principle if a pull request is opened.

**Comment by saltorre** [comment]
I am also `cd`ing into a reports directory first but it looks messy. I agree it would be great to be able to specify the output path.
